yarn-project/acir-simulator/src/avm/opcodes/.eslintrc.cjs

```diff
@@ -0,0 +1,8 @@
+const baseConfig = require('@aztec/foundation/eslint');
+module.exports = {
+  ...baseConfig,
+  rules: {
+    ...baseConfig.rules,
+    'require-await': 'off',
```

Comment:
I have added this eslint rule just for the opcodes folder to get around the opcodes being promises, I'm personally fine with them all being promises, but let me know if this is a symptom of "maybe we should start again"